WordPress/Classes/ViewRelated/Reader/ReaderHelpers.swift

```diff
@@ -49,8 +49,8 @@ struct ReaderPostMenuButtonTitles {
     static let unsubscribe = NSLocalizedString("Turn off site notifications", comment: "Verb. An option to switch off site notifications.")
     static let markSeen = NSLocalizedString("Mark as seen", comment: "An option to mark a post as seen.")
     static let markUnseen = NSLocalizedString("Mark as unseen", comment: "An option to mark a post as unseen.")
-    static let followConversation = NSLocalizedString("Follow conversation by email", comment: "Verb. Button title. Follow the comments on a post.")
-    static let unFollowConversation = NSLocalizedString("Unfollow conversation by email", comment: "Verb. Button title. The user is following the comments on a post.")
+    static let followConversation = NSLocalizedString("Follow Conversation", comment: "Verb. Button title. Follow the comments on a post.")
+    static let unFollowConversation = NSLocalizedString("Unfollow Conversation", comment: "Verb. Button title. The user is following the comments on a post.")
```

Comment:
I know I wrote "Follow Conversation", but after seeing it on a device, I realized that the other menu items are not capitalized. Can we change this to "Follow conversation" instead? Sorry about this! 😅 
```suggestion
    static let followConversation = NSLocalizedString("Follow conversation", comment: "Verb. Button title. Follow the comments on a post.")
    static let unFollowConversation = NSLocalizedString("Unfollow conversation", comment: "Verb. Button title. The user is following the comments on a post.")
```